sencha-workspace/SlateAdmin/app/controller/progress/interims/Report.js

```diff
@@ -310,21 +347,26 @@ Ext.define('SlateAdmin.controller.progress.interims.Report', {
 
     onSectionNotesSaveBtnClick: function() {
         var sectionNotesForm = this.getSectionNotesForm(),
-            sectionNotes = sectionNotesForm.getRecord();
+            sectionDataRecord = sectionNotesForm.getRecord(),
```

Comment:
@nbey rename to `sectionTermDataRecord` throughout for consistency, these models can get pretty confusing to follow if we're not absolutely consistent with naming them